xml/System.Windows.Forms/Application.xml

```diff
@@ -1975,8 +1976,8 @@ static class Program
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether visual styles are enabled for the application.</summary>
+        <value><see langword="true" /> if visual styles are enabled; otherwise, <see langword="false" />.</value>
         <remarks>To be added.</remarks>
```

Comment:
@RussKie should we say something here about when a mode wouldn't be set? Can we list the reasons?

Reply:
@mairaw feel free to edit my suggestion:

```suggestion
        <remarks>

## Remarks

The visual styles can be enabled by calling <xref:System.Windows.Forms.Application.EnableVisualStyles>.

The visual styles will not be enabled if the OS does not support them (which is unlikely), or theming is disabled at the OS level.


        </remarks>
```

Reply:
Many times our classic framework applications were targeted to windows 7  in the manifest file. VisualStyles are 'by default' enabled on windows 8 and above. For core, we do not have manifest.  So, need to check how this works/useful.

Reply:
@Tanya-Solyanik

Reply:
Some server SKUs  don't support visual styles, so I would remove `(which is unlikely)` clause.

Reply:
EnableVisualStyles and manifest file works fine in Core applicatins, here is a core app with disabled visual styles:
![image](https://user-images.githubusercontent.com/15823268/63044214-9832c200-be82-11e9-8d94-77a682da67f6.png)